d2.config.js

```diff
@@ -4,6 +4,8 @@ const config = {
     title: 'SMS Configuration',
     description: 'Configure SMS gateways and manage SMS messages',
     coreApp: true,
+    minDHIS2Version: '2.35',
+    id: '441cad0c-395a-4e93-8f10-475602397371',
```

Comment:
Do we have docs regarding to this?
I can find it in here: https://platform.dhis2.nu/#/config/d2-config-js-reference

Reply:
It should be added there for reference, but here is the full guide about continuous delivery https://developers.dhis2.org/docs/guides/publish-apphub